feat: Arrays and Loops The Odin Project (#54409)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/16-the-odin-project/top-learn-arrays-and-loops/learn-arrays-and-loops-question-b.md

---
id: 661e27568602567c118451d2
title: Learn Arrays and Loops Question B
challengeType: 15
dashedName: learn-arrays-and-loops-question-b
---
# --description--

To access the elements of an array, you can use the index number. The index number starts from 0, so the first element of an array is at index 0, the second element is at index 1, and so on.

For example, to access the first element of the `fruits` array, you can use the following code:

```javascript
const fruits = ['apple', 'banana', 'orange'];
console.log(fruits[0]); // Output: apple
console.log(fruits[2]); // Output: orange
```

If the index is out of range, JavaScript will return `undefined`. Meaning that if you try to access `fruits[3]` in the above example, it will return `undefined`.

# --question--

## --text--

> What is the index number of the fourth element in the `fruits` array?

Reviewer: ```suggestion
What is the element at the fourth index of the `fruits` array?
```

The fourth element doesn't exist and wouldn't have an index number.
